package.json

```diff
@@ -52,28 +52,28 @@
     "db:seed:prod": "node_modules/.bin/sequelize db:seed:all --options-path .production.sequelizerc",
     "db:seed:undo": "node_modules/.bin/sequelize db:seed:undo:all",
     "db:seed:undo:prod": "node_modules/.bin/sequelize db:seed:undo:all --options-path .production.sequelizerc",
-    "docker:deps": "docker-compose run --rm backend yarn global add node-gyp && docker-compose run --rm backend yarn install && docker-compose run --rm frontend yarn global add node-gyp && docker-compose run --rm frontend yarn install && docker-compose run --rm testingonly yarn global add node-gyp && docker-compose run --rm testingonly yarn install",
+    "docker:deps": "docker compose run --rm backend yarn global add node-gyp && docker compose run --rm backend yarn install && docker compose run --rm frontend yarn global add node-gyp && docker compose run --rm frontend yarn install && docker compose run --rm testingonly yarn global add node-gyp && docker compose run --rm testingonly yarn install",
     "docker:reset": "./bin/reset-all",
-    "docker:start": "docker-compose up",
-    "docker:start:debug": "docker-compose --compatibility -f docker-compose.yml -f docker-compose.debug.yml up",
-    "docker:stop": "docker-compose down",
-    "docker:dbs:start": "docker-compose -f 'docker-compose.yml' up",
-    "docker:dbs:stop": "docker-compose -f 'docker-compose.yml' down",
+    "docker:start": "docker compose up",
+    "docker:start:debug": "docker compose --compatibility -f docker compose.yml -f docker compose.debug.yml up",
```

Comment:
I think you went a little overzealous with ,your find/replace. the filename should still be docker-compose.yml